PACKAGES.md

```diff
@@ -0,0 +1,33 @@
+# Azure SDK for .NET Packages
+
```

Comment:
Would it better to put this information at https://azure.github.io/azure-sdk/releases/latest/dotnet.html?